manifests/base/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+images:
+  - name: lsstsqre/sherlock-frontend
+    newTag: 0.0.1
```

Comment:
You may want to keep this up to date with the expected release.

Reply:
Thanks updated